.chloggen/hostmetrics-add-mem-available.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

Comment:
Please add another `breaking` change log item for disabling slab states

Reply:
done